blackboxopt/visualizations/visualizer.py

```diff
@@ -127,42 +128,56 @@ def create_hover_information(sections: dict) -> Tuple[str, List]:
     return template, data_columns
 
 
-def multi_objective_visualization(evaluations: List[Evaluation]):
-    if not evaluations:
-        raise NoSuccessfulEvaluationsError
-
-    # Prepare dataframe for visualization
-    df = evaluations_to_df(evaluations)
+def _prepare_for_multi_objective_visualization(
+    df: pd.DataFrame, objectives: Sequence[Objective]
+) -> Tuple[pd.DataFrame, Dict[str, List[str]]]:
+    loss_matrix = np.array(
```

Comment:
What do you think about adding a comment here? Something along:

`# Unify potentially different objective directions (greater_is_better)`